src/emc/usr_intf/stepconf/stepconf.py

```diff
@@ -50,6 +50,11 @@
 import time
 from multifilebuilder_gtk3 import MultiFileBuilder
 
+try:
+    from defusedexpat import pyexpat as expat
+except ImportError:
+    from xml.parsers import expat
```

Comment:
this change seems unrelated to the main purpose of the PR. Can you try to separate these out more cleanly in the future?  (note: defusedexpat is not packaged in debian, but since it apparently degrades gracefully to the "normal" expat all is well)